src/test/groovy/graphql/validation/ValidationUtilTest.groovy

```diff
@@ -88,13 +88,13 @@ class ValidationUtilTest extends Specification {
         !validationUtil.isValidLiteralValue(arrayValue, type,schema)
     }
 
-    def "One value is a single element List"() {
-        given:
-        def singleValue = new BooleanValue(true)
-        def type = list(GraphQLBoolean)
-        expect:
-        validationUtil.isValidLiteralValue(singleValue, type,schema)
-    }
+//    def "One value is a single element List"() {
+//        given:
+//        def singleValue = new BooleanValue(true)
+//        def type = list(GraphQLBoolean)
+//        expect:
+//        validationUtil.isValidLiteralValue(singleValue, type,schema)
+//    }
```

Comment:
A test is needed.  I think a pure unit test and perhaps a integration tests from the top

Reply:
added unit test. thanks.